.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: ['plugin:react/recommended', 'standard'],
```

Comment:
This can't be ok. When you run npx eslint . from the terminal it raises an error because is not able to find that standard config.